11ty/documentation/development.md

```diff
@@ -5,44 +5,38 @@ parent:
+## Pre-requisites
 
-We are using [Eleventy](https://www.11ty.io/) to generate the site based on the given definitions. As a module bundle we use Parcel.
+To contribute to Self-Defined, you should be familiar with local development tools such as Git and the terminal, and comfortable working in a text editor such as Visual Studio Code, Sublime, or Atom. Additionally, you'll need to a GitHub account to submit your changes to us for review.
 
-## Setup
+We're working on making it possible to contribute definitions to Self-Defined without the need for developer tooling! Stay tuned!
 
-You will need to have Node and Git installed on your machine. To start developing please create a fork of the root repository.
+## Installing Self-Defined
 
-Afterwards, install the dependencies.
+In your terminal, run the following command:
 
 ```bash
 npm install
 ```
 
-That’s it, actually.
+This will install everything you need to run Self-Defined locally.
 
-## Development Mode
+## Starting the development server
 
-To start the dev server run this command:
+In your terminal, run the following command:
 
 ```bash
 npm run serve
 ```
 
-This will start Eleventy’s development server and watch the CSS entry point for changes. The output in the console will show the local URL to use.
+This will start a local development server. Once the startup process completes, you your terminal will show you the local URL that you can visit to view Self-Defined in your browser. We default to [http://localhost:8080](http://localhost:8080).
 
-### Eleventy Config
+The `serve` script also watches your files for changes, so that your browser will refresh while you work. Try opening `assets/css/abstracts/_variables.scss` and changing a color variable defined there, then save the file. You'll see your new color when you check your browser!
 
-Configuration for Eleventy (e.g. filters and collections) are defined in `.eleventy.js`.
+## About the project
 
-## Build
+We're using [Eleventy](https://www.11ty.io/), a static site generator, to generate the files that make up the Self-Defined website, and [Syntactically Awesome Stylesheets (SASS)](https://sass-lang.com) to write our CSS. You're welcome to make contributions to any part of this code base. If you're not sure where to begin, check out [our issues on GitHub](https://github.com/tatianamac/selfdefined/issues)! If you're having trouble, feel free to open an issue!
 
-To build the production site run
-
-```bash
-npm run build
-```
-
-This will generate a minified CSS bundle as well as run Eleventy’s production build. Build files are located in `dist`.
+Static site generators like Eleventy rely on something called _front matter_. Font matter provides Eleventy with useful information about what to do with a file as the site gets built. You can read about how we structured our [front matter for our definitions](/documentation/front-matter/) if you'd like to contribute a definition.
```

Comment:
@tatianamac:

🖊️When you proposed this addition, you wrote `Front Matter` and `Markdown` as proper nouns. I don't think they are proper nouns in parlance, so I didn't write them that way here. Let me know if I should change them.

Reply:
No, I was wearing a tie that day so I think it made me feel extra proper.

Lowercase is preferred!